src/components/Book.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
+
+const Book = ({ book }) => {
+  const { bookId, title, category } = book;
+  return (
+    <tr className = "book-row">
+      <td className = "book-column">{bookId}</td>
+      <td className = "book-column">{title}</td>
+      <td className = "book-column">{category}</td>
+    </tr>
+  );
+};
+
+Book.propTypes = {
+  book: PropTypes.objectOf.isRequired,
```

Comment:
- [ ] This proptypes raises a console error 
![Screenshot from 2020-12-29 12-14-45](https://user-images.githubusercontent.com/4246637/103276523-7aaabd80-49cf-11eb-9a3f-a25c1c9d3b65.png)

`objectOf` should accept a parameter, but Ideally we want to use `shape` instead

```js
book: PropTypes.shape({
    bookId: PropTypes.string.isRequired,
    title: PropTypes.string.isRequired,
    category: PropTypes.string.isRequired,
})
```

Kindly, check out [shape](https://reactjs.org/docs/typechecking-with-proptypes.html)